ec2uploadimg

```diff
@@ -704,10 +701,16 @@ def get_vpc_subnet_id(args, config, region, setup,  logger):
                 '--vpc-subnet-id'
             )
         except Exception:
+            # falls back to setup.create_vpc_subnet()
             msg = 'Not using a subnet-id, none given on the '
             msg += 'command line and none found in config for '
             msg += '"subnet_id_%s" value' % region
             logger.error(msg)
+
+        try:
+            vpc_subnet_id = vpc_subnet_id or setup.create_vpc_subnet()
```

Comment:
shouldn't we apply the same approach in line 696 ?

```python
vpc_subnet_id = vpc_subnet_id or utils.get_from_config(
```
as we do not know if the `vpc_subnet_id` is present or not at that point either